script.js

```diff
@@ -0,0 +1,9 @@
+menu.onclick = function myFunction() {
+    var x = document.getElementById('myTopnav');
+
+    if (x.className === "topnav") {
```

Comment:
Use soft tabs (space character) set to 2 spaces. eslint: indent
https://github.com/airbnb/javascript#whitespace--spaces